src/quick-demo/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { RainwayRuntime } from "rainway-sdk";
+import React, { useEffect, useRef, useState } from "react";
+import ReactDOM from "react-dom";
+import "../style.css";
+import useAsyncEffect from "use-async-effect";
+
+const QuickDemo = () => {
+  const [runtime, setRuntime] = useState<RainwayRuntime | undefined>();
+  const [error, setError] = useState<string>("");
+  useAsyncEffect(async (isAlive) => {
+    try {
+      const rt = await RainwayRuntime.initialize({
+        apiKey: "TODO",
```

Comment:
I both patched over the TODO and managed expectations in the README. (It might have really been a better idea to leave `quick-demo` entirely to another PR, but I didn't really want to tease apart the changes to the webpack config…)